CONTRIBUTING.md

```diff
@@ -36,10 +37,21 @@ Current setup: [ESLint](https://eslint.org) for `JavaScript` and [Black](https:/
 
 ### FAQ
 
+**Q:** I am new to the project, how do I view all available commands?
+**A:** Run `make help` in the directory containing a `Makefile`. Usage will be shown. Don't be afraid to read the file, it is quite strait forward and well documented.
 
-**Q:** Your pull request's CI is failing to push/pull images from Docker Hub?
+**Q:** My pull request's CI is failing to push/pull images from Docker Hub?
 **A:** Check your settings again, [see CI-notes](#ci-notes) and retry.
 
+**Q:** It is not clear how dependencies work. Where can I read more?
+**A:** [Management of python dependencies](docs/python-dependencies.md) has a detailed explanation and useful resources.
+
+**Q:** I see and `api` folder full of text files, what is it?
+**A:** Look at [APIs development guidelines](api/README.md) it will point you in the right direction.
+
+**Q:** Are there any special code conventions?
+**A:** Just a few [oSPARC Coding Conventions](docs/coding-conventions.md).
```

Comment:
these coding conventions probably are absolutely outdated! I would rather remove them and point to the linter configuration files or commands (e.g. for python pylint and black; and for javascripts eslint)

Reply:
I am talking with @odeimaiz for updates. I will figure out how to point people in the right direction from that document.

Reply:
Ok I've changed it a bit. You could have a look at it now.